diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9d28ae3..a47eec0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -793,6 +793,10 @@ if(AVIF_BUILD_APPS)
         set_target_properties(avifdec PROPERTIES LINKER_LANGUAGE "CXX")
     endif()
     target_link_libraries(avifdec avif_apps)
+    if(WIN32 AND MINGW)
+	    target_link_options(avifenc PUBLIC "SHELL:-municode")
+	    target_link_options(avifdec PUBLIC "SHELL:-municode")
+    endif()
 
     if(NOT SKIP_INSTALL_APPS AND NOT SKIP_INSTALL_ALL)
         install(
