diff --git a/src/codec_svt.c b/src/codec_svt.c
index 986c3d1..262487e 100644
--- a/src/codec_svt.c
+++ b/src/codec_svt.c
@@ -115,11 +115,7 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
         // See https://gitlab.com/AOMediaCodec/SVT-AV1/-/issues/1697.
         memset(svt_config, 0, sizeof(EbSvtAv1EncConfiguration));
 
-#if SVT_AV1_CHECK_VERSION(3, 0, 0)
         res = svt_av1_enc_init_handle(&codec->internal->svt_encoder, svt_config);
-#else
-        res = svt_av1_enc_init_handle(&codec->internal->svt_encoder, NULL, svt_config);
-#endif
         if (res != EB_ErrorNone) {
             goto cleanup;
         }
@@ -131,11 +127,7 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
 #endif
         svt_config->source_width = image->width;
         svt_config->source_height = image->height;
-#if SVT_AV1_CHECK_VERSION(3, 0, 0)
         svt_config->level_of_parallelism = encoder->maxThreads;
-#else
-        svt_config->logical_processors = encoder->maxThreads;
-#endif
         svt_config->enable_adaptive_quantization = 2;
         // disable 2-pass
 #if SVT_AV1_CHECK_VERSION(0, 9, 0)
