diff --git a/examples/unicode.h b/examples/unicode.h
index 1a8581c..9645727 100644
--- a/examples/unicode.h
+++ b/examples/unicode.h
@@ -25,9 +25,9 @@
 
 #include <fcntl.h>
 #include <io.h>
-#include <shellapi.h>
 #include <wchar.h>
 #include <windows.h>
+#include <shellapi.h>
 
 // Create a wchar_t array containing Unicode parameters.
 #define INIT_WARGV(ARGC, ARGV)                                       \
