diff --git a/filters/f_lavfi.c b/filters/f_lavfi.c
index 597cf91fc0..51fa5369a1 100644
--- a/filters/f_lavfi.c
+++ b/filters/f_lavfi.c
@@ -1024,7 +1024,11 @@ static const char *get_avopt_type_name(enum AVOptionType type)
     case AV_OPT_TYPE_VIDEO_RATE:        return "fps";
     case AV_OPT_TYPE_DURATION:          return "duration";
     case AV_OPT_TYPE_COLOR:             return "color";
-    case AV_OPT_TYPE_CHANNEL_LAYOUT:    return "channellayout";
+#if LIBAVUTIL_VERSION_MAJOR < 59
+    case AV_OPT_TYPE_CHANNEL_LAYOUT:    return "ch_layout";
+#else
+    case AV_OPT_TYPE_CHLAYOUT:    	return "ch_layout";
+#endif
     case AV_OPT_TYPE_BOOL:              return "bool";
     case AV_OPT_TYPE_CONST: // fallthrough
     default:
