diff --git a/CMakeLists.txt b/CMakeLists.txt
index 48953de5..94f3e26f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -287,7 +287,7 @@ macro(add_clike_and_ld_flags)
 endmacro()
 
 
-option(SVT_AV1_LTO "Attempt to enable Link Time Optimization if available" OFF)
+option(SVT_AV1_LTO "Attempt to enable Link Time Optimization if available" ON)
 if(SVT_AV1_LTO)
     if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.9)
         include(CheckIPOSupported)
