diff --git a/ogg123/ogg123.c b/ogg123/ogg123.c
index 3e56aba..58ccd7e 100644
--- a/ogg123/ogg123.c
+++ b/ogg123/ogg123.c
@@ -512,7 +512,9 @@ int main(int argc, char **argv)
 
   ao_onexit (options.devices);
 
+#ifndef _WIN32
   convert_free_charset();
+#endif
   exit (exit_status);
 }
 
diff --git a/oggenc/oggenc.c b/oggenc/oggenc.c
index a8fd54f..367415b 100644
--- a/oggenc/oggenc.c
+++ b/oggenc/oggenc.c
@@ -499,7 +499,9 @@ clear_all:
         }
     }/* Finished this file, loop around to next... */
 
+#ifndef _WIN32
     convert_free_charset();
+#endif
     return errors?1:0;
 
 }
diff --git a/ogginfo/ogginfo2.c b/ogginfo/ogginfo2.c
index ef9739f..45b6e44 100644
--- a/ogginfo/ogginfo2.c
+++ b/ogginfo/ogginfo2.c
@@ -466,6 +466,8 @@ int main(int argc, char **argv) {
             ret = flawed;
     }
 
+#ifndef _WIN32
     convert_free_charset();
+#endif
     return ret;
 }
