diff --git a/source/common/x86/x86inc.asm b/source/common/x86/x86inc.asm
index 974980c..184d7ea 100644
--- a/source/common/x86/x86inc.asm
+++ b/source/common/x86/x86inc.asm
@@ -91,7 +91,7 @@
     %endif
 %endmacro
 %macro SECTION_IBT_SHSTK 0
-  %if ENABLE_CET
+  %ifdef ENABLE_CET
     %ifidn __OUTPUT_FORMAT__,win32
     %elif WIN64
     %else
@@ -760,7 +760,7 @@ BRANCH_INSTR jz, je, jnz, jne, jl, jle, jnl, jnle, jg, jge, jng, jnge, ja, jae,
     %assign stack_size 0        ; amount of stack space that can be freely used inside a function
     %assign stack_size_padded 0 ; total amount of allocated stack space, including space for callee-saved xmm registers on WIN64 and alignment padding
     %assign xmm_regs_used 0     ; number of XMM registers requested, used for dealing with callee-saved registers on WIN64 and vzeroupper
-    %if ENABLE_CET
+    %ifdef ENABLE_CET
       %if ARCH_X86_64
         endbr64
       %else
