diff --git a/source/CMakeLists.txt b/source/CMakeLists.txt
index 13e4750de..733b9dab9 100755
--- a/source/CMakeLists.txt
+++ b/source/CMakeLists.txt
@@ -25,7 +25,7 @@ include(CheckCXXCompilerFlag)
 
 option(FPROFILE_GENERATE "Compile executable to generate usage data" OFF)
 option(FPROFILE_USE "Compile executable using generated usage data" OFF)
-option(NATIVE_BUILD "Target the build CPU" OFF)
+option(NATIVE_BUILD "Target the build CPU" ON)
 option(STATIC_LINK_CRT "Statically link C runtime for release builds" OFF)
 mark_as_advanced(FPROFILE_USE FPROFILE_GENERATE NATIVE_BUILD)
 # X265_BUILD must be incremented each time the public API is changed
